packages/go_router/lib/src/route.dart

```diff
@@ -114,6 +114,11 @@ abstract class RouteBase {
 /// The Page depends on the application type: [MaterialPage] for
 /// [MaterialApp], [CupertinoPage] for [CupertinoApp], or
 /// [NoTransitionPage] for [WidgetsApp].
+///
+/// {@category Get started}
```

Comment:
This is cool

Reply:
Shout out to @jonasfj for the suggestion!